src/clib/lib/enkf/ext_param.cpp

```diff
@@ -212,6 +213,19 @@ ext_param_config_type const *ext_param_get_config(const ext_param_type *param) {
 
 VOID_ALLOC(ext_param)
 VOID_FREE(ext_param)
-VOID_ECL_WRITE(ext_param)
 VOID_WRITE_TO_BUFFER(ext_param)
 VOID_READ_FROM_BUFFER(ext_param)
+
+ERT_CLIB_SUBMODULE("ext_param", m) {
+    m.def("ecl_write",
```

Comment:
This name is not very descriptive, can you rename it to something else? For example: `write_parameter_to_file`?